ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/post_workload.yml

```diff
@@ -26,4 +24,4 @@
     msg: "Post-Software checks completed successfully"
   when:
   - not silent|bool
-  - workload_shared_deployment|default(False)
\ No newline at end of file
+  - workload_shared_deployment|default(False)
```

Comment:
Same here:
```
  - workload_shared_deployment | default(false) | bool
```

Reply:
Haven't changed things like that anywhere. Will look over this and make the changes.